bin/crew

```diff
@@ -51,6 +51,13 @@ require_gem 'pry-byebug' if CREW_DEBUG
 # being started with a --debug flag.
 # The following line can be used anywhere as a breakpoint.
 binding.pry if CREW_DEBUG
+
+begin
+  require_relative '../lib/crew_mvdir'
```

Comment:
This feels unusual-- why don't we just install it into `CREW_PREFIX/lib` or wherever is appropriate and search for it using `require`?

Reply:
Not sure if that's a good place for it 😅 (putting it there just like other `.rb` libraries for `crew`)

Btw I am okay with moving it to `CREW_LIB_PREFIX`...

Reply:
How about `#{CREW_PREFIX}/share/crew_mvdir`?

Reply:
Given that this is a compiled library, I was thinking it made more sense to treat it like any other package.